src/modules/layerManager/components/LayerItem.js

```diff
@@ -126,7 +138,7 @@ export class LayerItem extends AbstractMvuContentPanel {
 
 		const duplicateLayer = () => {
 			//state store change -> implicit call of #render()
-			addLayer(`${layer.geoResourceId}_${createUniqueId()}`, { ...layer, label: `${layer.label} (${translate('layerManager_layer_copy')})`, zIndex: layer.zIndex + 1 });
+			addLayer(`${layer.geoResourceId}_${createUniqueId()}`, { ...layer, geoResourceId: layer.geoResourceId, label: `${layer.label} (${translate('layerManager_layer_copy')})`, zIndex: layer.zIndex + 1 });
```

Comment:
We should verify having a layer object containing a geoResourceId property in the store